core/controllers/blog_admin.py

```diff
@@ -116,8 +116,13 @@ def post(self) -> None:
             logging.info(
                 '[BLOG ADMIN] %s saved config property values: %s' %
                 (self.user_id, new_config_property_values))
-        elif self.normalized_payload.get(
-                'action') == 'revert_config_property':
+        else:
+            # The handler schema defines the possible values of 'action'.
+            # If 'action' has a value other than those defined in the schema,
+            # a Bad Request error will be thrown. Hence, 'action' must be
+            # 'revert_config_property' if this branch is executed.
+            assert self.normalized_payload.get(
+                'action') == 'revert_config_property'
```

Comment:
Please create `action` variable above the `if` so then you can do. 

```suggestion
            assert action == 'revert_config_property'
```

Reply:
Done